testing/020/indexpage.xml

```diff
@@ -6,9 +6,21 @@
     <briefdescription>
     </briefdescription>
     <detaileddescription>
-      <para>Text.       
+      <para>Text. <htmlonly> 
+HTML
+</htmlonly> <htmlonly block="yes">
+HTML with block
+</htmlonly> <rtfonly> 
```

Comment:
Thanks for doing this! 👍 

One "stretch goal" -- the documentation is stating this:

> When you want to insert a HTML fragment that has block scope like a table or list which should appear outside `<p>..</p>`, this can lead to invalid HTML. You can use `\htmlonly[block]` to make doxygen end the current paragraph and restart it after `\endhtmlonly`.

Ideally that would hold also for the XML output, so either `<htmlonly>` being inside a `<para>`, or `<htmlonly>` *not* being wrapped in a `<para>` (instead of current `<htmlonly block="yes">` being inside a `<para>`). Basically so it doesn't need to be extracted out of the paragraph on the client side. 

How hard would it be to implement it like this?

Reply:
In other words, the output looking like this instead:

```xml
<para>Text. <htmlonly> 
HTML
</htmlonly></para>
<htmlonly block="yes">
HTML with block
</htmlonly>
<para><rtfonly> 
RTF
</rtfonly> <manonly> 
Man
</manonly> <latexonly> 
LaTeX
</latexonly>
```

Reply:
No idea at the moment how difficult this would be and whether this would be conceptual OK (as it is not really a "dump" anymore).
When really necessary I think best is to submit an issue for this as soon as this proposed pull request has been integrated in master on github.

Reply:
Okay, no problem. Asked only in case it would be easy to do to do on your side -- I can do the paragraph extraction on my side as well ;)